dom.bs

```diff
@@ -894,8 +894,8 @@ for historical reasons.
 <pre class=idl>
 [Constructor, Exposed=(Window,Worker)]
 interface EventTarget {
-  void addEventListener(DOMString type, EventListener? callback, optional (AddEventListenerOptions or boolean) options);
-  void removeEventListener(DOMString type, EventListener? callback, optional (EventListenerOptions or boolean) options);
+  void addEventListener(DOMString type, EventListener? callback, optional (AddEventListenerOptions or boolean) options = false);
```

Comment:
Actually, maybe this isn't needed because omitting the argument means an implicit dictionary gets passed. I'm not a 100% sure that's also true if the dictionary is part of a union, but it'd make sense.